test/testdata/custom_column_types.yml

```diff
@@ -26,5 +26,5 @@ tables:
   - name: "FullTypes"
     columns:
       FTInt: "int32"
-      FTInitNull: "uint64"
+      FTIntNull: "uint64"
```

Comment:
I've fixed this typo then it raises an error since yo tries to convert `spanner.NullInt64` into `uint64`. What should I do to resolve the problem?

https://github.com/cloudspannerecosystem/yo/pull/84/commits/3790e3bd102579a3dfc37f22d5a467af4e64b02f#diff-a39ac8c53baa368da95a760a15e95e386ee6ee9ec40d7eed3240717102f68ec0R291

Reply:
hmm, it seems broken in this type conversion. Please just remove this line from the yaml for now.